app/docs/0.11.x/clustering.md

```diff
@@ -4,54 +4,317 @@ title: Clustering Reference
+
+### Interacting with the cache via the Admin API
+
+If for some reason, you wish to investigate the cached values, or manually
+invalidate a value cached by Kong (a cached hit or miss), you can do so via the
+Admin API `/cache` endpoint.
+
+#### Inspect a cached value
+
+##### Endpoint
+
+<div class="endpoint get">/cache/{cache_key}</div>
+
+##### Response
+
+If a value with that key is cached:
+
+```
+HTTP 200 OK
+...
+{
+    ...
+}
+```
+
+Else:
+
+```
+HTTP 404 Not Found
+```
+
+**Note**: retrieving the `cache_key` for each entity being cached by Kong is
+currently an undocumented process. Future versions of the Admin API will make
+this process easier.
+
+[Back to TOC](#table-of-contents)
 
-A Kong cluster can be created in one datacenter, or in multiple datacenters,
-in both cloud or on-premise environments. Kong will take care of all cluster
-related synchronisation automatically, as long as the nodes are configured
-to use the same datastore.
+#### Purge a cached value
 
-## Summary
+##### Endpoint
 
-- 1. [Getting Started][1]
-- 2. [Network Requirements][2]
+<div class="endpoint delete">/cache/{cache_key}</div>
 
-[1]: #1-getting-started
-[2]: #2-network-requirements
+##### Response
 
-## 1. Getting Started
+```
+HTTP 204 No Content
+...
+```
 
-Kong nodes pointing to the same datastore will join in the same Kong cluster.
-Changes (to for example APIs, Consumers, Credentials, etc...) are propagated
-over the Kong cluster through events that are being
-registered in the datastore. Periodically all nodes will check for updates
-here and apply the changes.
+**Note**: retrieving the `cache_key` for each entity being cached by Kong is
+currently an undocumented process. Future versions of the Admin API will make
+this process easier.
 
-Kong clustering settings are specified in the configuration file via the
-following entries:
+[Back to TOC](#table-of-contents)
 
-* [db_update_frequency][db_update_frequency]
-* [db_update_propagation][db_update_propagation]
+#### Purge a node's cache
 
-<div class="alert alert-warning">
-  Please note that Kong follows an `eventual consistency` model and hence
-  there will be some latency for changes to be effectuated across the cluster.
-</div>
+##### Endpoint
 
-## 2. Network Requirements
+<div class="endpoint delete">/cache</div>
 
-There are no networking requirements other than Kong being able to connect to
-the datastore.
+##### Response
 
-When multiple datacenters or regions are being used (or any other reason to
-suspect latency in datastore updates) then please adjust the
-[db_update_propagation][db_update_propagation] setting to match accordingly.
+```
+HTTP 204 No Content
+```
 
+**Note**: be wary of using this endpoint on a warm, production running node.
+If the node is receiving a lot of traffic, purging its cache at the same time
+will trigger many requests to your database, and could cause a
+[dog-pile effect](https://en.wikipedia.org/wiki/Cache_stampede).
```

Comment:
It could still be a lot of different entities at once that are not cached. The behavior could still have a certain impact on performance and on the DB. Better safe than sorry?